File: 프로그래머스/0/181878. 원하는 문자열 찾기/원하는 문자열 찾기.java
class Solution {
    public int solution(String myString, String pat) {
        String new_string = "";
        String new_pat = "";
        for(char i:myString.toCharArray()){
            new_string+=Character.toLowerCase(i);
        }
        for(char i:pat.toCharArray()){
            new_pat+=Character.toLowerCase(i);
        }
        return new_string.contains(new_pat)?1:0;
    }
}

File: 프로그래머스/0/181878. 원하는 문자열 찾기/원하는 문자열 찾기.kt
class Solution {
    fun solution(myString: String, pat: String): Int {
        if (myString.length < pat.length) {
            return 0
        }
        
        val temp1 = myString.toLowerCase()
        val temp2 = pat.toLowerCase()
        
        return if(temp2 in temp1) 1 else 0
    }
}